projects/dsp-ui/src/lib/viewer/views/list-view/list-view.component.spec.ts

```diff
@@ -78,11 +69,15 @@ describe('ListViewComponent', () => {
         TestBed.configureTestingModule({
             declarations: [
                 ListViewComponent,
+                ResourceListComponent,
+                ResourceGridComponent,
                 TestParentComponent
             ],
             imports: [
+                DspActionModule,
```

Comment:
Could you mock the things you need from the DSPActionModule?

In general, importing our own code in unit tests should be avoided (however, sometimes there is a good reason for not using a mock).

Reply:
Done in d4c83b0